tests/integration/kubernetes/k8s-guest-pull-image-encrypted.bats

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bats
+# Copyright (c) 2024 IBM Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/confidential_kbs.sh"
+
+export KBS="${KBS:-false}"
+
+setup() {
+    if is_confidential_hardware; then
+        skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
```

Comment:
Hi @stevenhorsman !

Is this skip intentional or just inherited from non-encrypted tests (k8s-guest-pull-image.bats)?

Reply:
A bit of both. It seems like the pattern we are taking (see guest-pull, attestation and sealed-secrets tests) is to first enabled it on non-TEE and then to test and roll it out to the other runtimeclasses, in order to not be delayed but issues with just one platform. I'm happy to test it on all now in case though

Reply:
So the SNP tests passed for encrypted image, but other failed and I'm not sure if it was related or now.
TDX failed, but I'm not sure why as the behaviour lots correct

Reply:
Got it. I only wanted to check it was intentional. IMHO we should stick with the tradition here: enable for no-tee then roll it out to the TEEs as follow ups.